fdbserver/workloads/BlobGranuleCorrectnessWorkload.actor.cpp

```diff
@@ -249,6 +249,8 @@ struct BlobGranuleCorrectnessWorkload : TestWorkload {
 		}
 	}
 
+	void disableFailureInjectionWorkloads(std::set<std::string>& out) const override { out.insert("Attrition"); }
```

Comment:
These aren't necessary after I fixed all of the BlobGranule attrition bugs

Reply:
cool! I'll remove this